docs/integrations/partykit.md

```diff
@@ -108,12 +108,10 @@ export default {
         room.broadcast(
           JSON.stringify({
             type: REFRESH_TYPE,
-          })
+          }),
         );
 
-        return new Response(
-          `Message sent to ${room.connections.size} connected participants`
-        );
+        return new Response(`Message sent to ${room.connections.size} connected participants`);
```

Comment:
this was intentionally kept in a new line because it helps the code block on generated website appear nicely without any horizontal scrolling.